test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -1,258 +1,247 @@
-			@{ testName = "null"       ; testObject = $null }
-		) {
-			param ($testObject)
-				$result = $testObject | Format-Table -Property "foo","bar" | Out-String
-				$result | Should -BeNullOrEmpty
-		}
-
-		It "Format-Table with '<testName>' string and -Force should output table with requested properties" -TestCases @(
-			@{ testName = "single line"; testString = "single line string" },
-			@{ testName = "multi line" ; testString = "line1`nline2"       },
-			@{ testName = "array"      ; testString = "line1","line2"      }
-		) {
-			param ($testString)
-			$result = $testString | Format-Table -Property "foo","bar" -Force | Out-String
-			$result.Replace(" ","").Replace([Environment]::NewLine,"") | Should -BeExactly "foobar------"
-		}
-
-		It "Format-Table with complex object for End-To-End should work" {
-				Add-Type -TypeDefinition "public enum MyDayOfWeek{Sun,Mon,Tue,Wed,Thu,Fri,Sat}"
-				$eto = New-Object MyDayOfWeek
-				$info = @{}
-				$info.intArray = 1,2,3,4
-				$info.arrayList = "string1","string2"
-				$info.enumerable = [MyDayOfWeek]$eto
-				$info.enumerableTestObject = $eto
-				$result = $info|Format-Table|Out-String
-				$result | Should -Match "intArray\s+{1, 2, 3, 4}"
-				$result | Should -Match "arrayList\s+{string1, string2}"
-				$result | Should -Match "enumerable\s+Sun"
-				$result | Should -Match "enumerableTestObject\s+Sun"
-		}
-
-		It "Format-Table with Expand Enumerable should work" {
-				$obj1 = "x 0","y 0"
-				$obj2 = "x 1","y 1"
-				$objs = New-Object System.Collections.ArrayList
-				$objs.Add($obj1)
-				$objs.Add($obj2)
-				$mo = [PSCustomObject]@{name = "this is name";sub = $objs}
-				$result1 = $mo|Format-Table -Expand CoreOnly|Out-String
-				$result1 | Should -Match "name\s+sub"
-				$result1 | Should -Match "this is name"
-
-				$result2 = $mo|Format-Table -Expand EnumOnly|Out-String
-				$result2 | Should -Match "name\s+sub"
-				$result2 | Should -Match "this is name\s+{x 0 y 0, x 1 y 1}"
-
-				$result3 = $mo|Format-Table -Expand Both|Out-String
-				$result3 | Should -Match "name\s+sub"
-				$result3 | Should -Match "this is name\s+{x 0 y 0, x 1 y 1}"
-		}
-
-		It "Format-Table should not add trailing whitespace to the header" {
-			$out = "hello" | Format-Table -Property foo -Force | Out-String
-			$out.Replace([System.Environment]::NewLine, "") | Should -BeExactly "foo---"
-		}
-
-		It "Format-Table should not add trailing whitespace to rows" {
-			$out = [pscustomobject]@{a=1;b=2} | Format-Table -HideTableHeaders | Out-String
-			$out.Replace([System.Environment]::NewLine, "") | Should -BeExactly "1 2"
-		}
-
-		It "Format-Table should have correct alignment" {
-			$ps1xml = @"
+        It "Should call format table on piped input without error" {
+                { Get-Date | Format-Table } | Should -Not -Throw
+        }
+
+        It "Should return a format object data type" {
+                $val = (Get-Date | Format-Table | Get-Member )
```

Comment:
Add space between open parenthesis and `Get-Date`

Reply:
Actually, the parenthesis seems unnecessary, can we remove it?

Reply:
Removed all extra parentheses.